packages/api-docs-builder/utils/replaceUrl.ts

```diff
@@ -18,7 +18,7 @@ export const replaceMaterialLinks = (url: string) => {
     return url;
   }
   return url.replace(
-    /(guides|customization|getting-started|discover-more|experimental-api|migration)/,
+    /(guides|customization|getting-started|discover-more|experimental-api|migration|integrations)/,
```

Comment:
Add this change to fix the wrong generated url of the deploy preview

<img width="718" alt="image" src="https://github.com/user-attachments/assets/c22f337e-d96f-4d31-9d48-9c6e9678fa9c" />

In the image above, the `material` should be `material-ui`.